sstable/colblk_writer.go

```diff
@@ -1002,6 +1007,19 @@ func (w *RawColumnWriter) Close() (err error) {
 		w.props.ValueBlocksSize = vbStats.ValueBlocksAndIndexSize
 	}
 
+	// Write the blob reference index block if non-empty.
+	if len(w.blobRefLivenessIndexBlock.bufs) > 0 {
```

Comment:
nit: thoughts on exposing a `NumReferences()` method and calling it here to make this conditional a little clearer?